### ISSUE TITLE
Command line parameter "email" does not work any more in "Pilot"

### ISSUE THREAD

**Issue by dg76** [fastlane/fastlane#4043]
### Issue Description

The command line parameters "-e" and "--email" do not seem to work any more, pilot still asks for the tester's email despite it was specified with "-e":

pilot add -p 12345678 -u user@test.com -f "First" -l "Last" -e "email@email.com" 
Email address of the tester:

This worked fine in previous versions of "pilot".

### Environment

pilot 1.4.1

**Comment by TKBurner** [comment]
@dg76 Thanks for reaching out. Can you send me the full output of when you run  pilot? I want to have a look. Thanks!

**Comment by dg76** [comment]
There is not much interesting output. After I enter the email address manually the rest seems to work fine and the tester also appears in iTunes connect:

$ pilot add -p 12345678 -u user@test.com -f "First" -l "Last" -e "email@email.com" 
Email address of the tester: email@email.com _<== I had to enter the email address here manually again_
[14:34:02]: Login to iTunes Connect (user@test.com)
[14:34:06]: Login successful
[14:34:09]: Successfully invited tester: email@email.com
[14:34:10]: Successfully added tester to app 12345678

As far as I can see only the command line parameter does not work any more.

**Comment by mfurtak** [comment]
Hi @dg76 - thanks for the report! :+1: 

I just ran your command but I wasn't prompted the way you were.
```
$ pilot add -u my@email.com -f "First" -l "Last" -e "email@email.com"
[13:31:35]: Login to iTunes Connect (my@email.com)
[13:31:38]: Login successful
[13:31:39]: Successfully invited tester: email@email.com
[13:31:40]: Successfully added tester to app com.myapp.App
```

You mentioned that you are on `pilot` **1.4.1**. I suspect you already have looked, but can you double-check by running `pilot -v`?

If you leave the `-e` argument off and just let the email address be the trailing parameter, does that work? Like: `$ pilot add -u my@email.com -f "First" -l "Last" "email@email.com"`

**Comment by dg76** [comment]
What I wrote was already the output of "pilot -v":

```
$ pilot -v
pilot 1.4.1
```

That's strange that you don't get this prompt. But you are right, without the "-e" it works fine. But why? Why does it work differently on my computer than on yours?

**Comment by mfurtak** [comment]
Aha! I have figured out what is going on.

I had installed `pilot` from my local development environment which contains a fix for this problem. However, this fix has not yet been released!

I am planning to do a release of `pilot` today, so this fix should be available soon. I'll update you here when it is available. :+1:

**Comment by dg76** [comment]
Great! :+1: Then please apologize my bug report. I had already waited a few days for a fix but then I thought I should check the issues list here. And when I found no similar report I thought no one would have seen this problem yet.

I am glad that it is already fixed in the next version! It is not urgent, I can just omit the "-e" until it is fixed. Thank you very much for your fast replies!

**Comment by mfurtak** [comment]
No apology needed! I'm glad to help :smile:

**Comment by mfurtak** [comment]
Hi @dg76 - we've just released [`pilot` 1.5.0](https://github.com/fastlane/fastlane/releases/tag/pilot%2F1.5.0). Please update and let me know if `-e` works as you would expect!

**Comment by dg76** [comment]
Yes, it seems to work now. Thank you very much! :+1:  😊
